webapps-backup/src/main/java/io/camunda/webapps/backup/repository/elasticsearch/ElasticsearchBackupRepository.java

```diff
@@ -361,24 +365,23 @@ public boolean isSnapshotFinishedWithinTimeout(
 
   private boolean snapshotWentWell(final SnapshotInfo snapshotInfo) {
     if (snapshotInfo.state() == SUCCESS) {
-      LOGGER.info("Snapshot done: " + snapshotInfo.snapshotId());
+      LOGGER.info("Snapshot done: {}", snapshotInfo.snapshotId());
       return true;
     } else if (snapshotInfo.state() == FAILED) {
       LOGGER.error(
-          String.format(
-              "Snapshot taking failed for %s, reason %s",
-              snapshotInfo.snapshotId(), snapshotInfo.reason()));
+          "Snapshot taking failed for {}, reason {}",
+          snapshotInfo.snapshotId(),
+          snapshotInfo.reason());
       // No need to continue
       return false;
     } else {
       LOGGER.warn(
-          String.format(
-              "Snapshot state is %s for snapshot %s",
-              snapshotInfo.state(), snapshotInfo.snapshotId()));
+          "Snapshot state is {} for snapshot {}", snapshotInfo.state(), snapshotInfo.snapshotId());
       return false;
     }
   }
 
+  // Check: done with differences see ChecK: BELOW
```

Comment:
:question: Are all these checks todos? :smile:

Reply:
They are mostly marks for the few differences between tasklist & operate implementation. I'll remove them when removing the impl in tasklist :+1: